octobot_trading/personal_data/orders/order_util.py

```diff
@@ -335,3 +333,24 @@ async def _cancel_reduce_only_orders_on_position_reset(exchange_manager, symbol)
             await order.trader.cancel_order(order)
             if order.order_group:
                 await order.order_group.on_cancel(order)
+
+
+def get_order_quantity_currency(exchange_manager, symbol, side=enums.PositionSide.BOTH):
+    try:
+        base, quote = symbol_util.split_symbol(symbol)
+    except ValueError:
+        # symbol that can't be split
+        return None
+    if exchange_manager.is_future:
+        if not exchange_manager.exchange.get_pair_future_contract(symbol).is_one_way_position_mode():
+            # side might change here
+            logging.get_logger(LOGGER_NAME).error("Not one way position mode are not implemented in "
+                                                  "get_order_quantity_currency")
+        else:
+            side = enums.PositionSide.BOTH
```

Comment:
Why do we override the side param value here?

Reply:
Order already has a attribute named `position_side`

Reply:
should we use `order.get_position_side()` ?

Reply:
Yes, passed as `side` value